desktop/core/src/desktop/lib/scheduler/api.py

```diff
@@ -15,25 +15,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import logging
 import sys
+import logging
 
 from django.forms.formsets import formset_factory
 from django.urls import reverse
 
 from desktop.auth.backend import is_admin
-from desktop.conf import TASK_SERVER
-from desktop.models import Document2
+from desktop.conf import TASK_SERVER_V2
```

Comment:
This change is not needed because we never use TASK_SERVER in oozie.

Reply:
Removed this import